frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/FinallyNode.tsx

```diff
@@ -0,0 +1,131 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { chart_color_blue_300 as blueColor } from '@patternfly/react-tokens';
+import { observer, Node, Layer } from '@patternfly/react-topology';
+import { pipelineRunFilterReducer } from '../../../utils/pipeline-filter-reducer';
+import { PipelineVisualizationTask } from '../detail-page-tabs/pipeline-details/PipelineVisualizationTask';
+import {
+  BUILDER_NODE_ADD_RADIUS,
+  NODE_WIDTH,
+  NODE_HEIGHT,
+  FINALLY_NODE_PADDING,
+  FINALLY_NODE_VERTICAL_SPACING,
+  FINALLY_ADD_LINK_TEXT_HEIGHT,
+  BUILDER_NODE_ERROR_RADIUS,
+  FINALLY_ADD_LINK_SIZE,
+} from './const';
+import PlusNodeDecorator from './PlusNodeDecorator';
+import ErrorNodeDecorator from './ErrorNodeDecorator';
+import TaskList from './TaskList';
+
+const FinallyNode: React.FC<{ element: Node }> = ({ element }) => {
+  const {
+    clusterTaskList = [],
+    namespaceTaskList = [],
+    task,
+    pipeline,
+    pipelineRun,
+    disableBuilder,
+  } = element.getData();
+  const { t } = useTranslation();
+  const { finallyTasks = [], finallyListTasks = [] } = task;
+  const FINALLY_TASK_NODE_HEIGHT = NODE_HEIGHT + FINALLY_NODE_VERTICAL_SPACING;
+  const allTasksLength = finallyTasks.length + finallyListTasks.length;
+  return (
+    <Layer id="groups">
```

Comment:
Perhaps I am missing something RE: Topology... why the Layer? Nothing else in Pipeline Topology has or needs a layer, this "node" is a glorified container for sure but it's self contained and there is no drag/drop sorta functionality...

Reply:
Not required, It was leftover from my initial layout grouping approach, I removed it now.